### ISSUE TITLE
Android Basics issue:

### ISSUE THREAD

**Issue by viramente** [google-developer-training/android-basics-kotlin-create-dice-roller-with-button-app-solution#1]
**URL of codelab**
https://developer.android.com/codelabs/basic-android-kotlin-training-dice-roller-images#0

**In which task and step of the codelab can this issue be found?**
2.1

**Describe the problem**
I have downloaded and opened the project from Github. The app should look like at the lesson, but it doesn't, there is a message on grey background: "Design editor is unavailable until next gradle sync." 
I'm beginner and I don't know what does it mean and how to resolve the problem and build the project.

**Steps to reproduce?**
1. Go to...
2. Click on...
3. See error...

**Versions**
_Android Studio version:_ 4.0.1
_API version of the emulator:_ 

**Additional information**
_Include screenshots if they would be useful in clarifying the problem._
![printscrean01](https://user-images.githubusercontent.com/69940079/96353401-f2a16100-10cb-11eb-9170-a11b54c8fddb.jpg)

**Comment by jtavva** [comment]
@viramente 
Thank you for reaching out to us.
To resolve your issue: 

1.  In Android Studio, select  **File** ->  **Sync Project with Gradle Files**.

![image](https://user-images.githubusercontent.com/24260141/96626871-e6ccbf00-12c4-11eb-8ae3-9bbff2797ab8.png)






If that does't work do the following:
1. Select **Build** -> **Clean Project**.
2. Select **Build** -> **Rebuild Project**

**Comment by viramente** [comment]
Thank you for your answer!
But I don't have such textlines in my Android Studio (I've attached the printscreens).

I've resolved the problem:
1) I opened the Project in Android Studio and got the message "Design editor is unavailable until next gradle sync." in app\src\main\res\layout\activity_main.xml.
2) I closed the project.
3) There was formed the folder ".idea" in the opened folder.
4) I cut all files from folder "DiceRoller" and all files out of this folder -- and then pasted into the folder containing the folder ".idea" with replacing all existing files with the same names.
5) I opened the project in Android Studio again and saw in the right bottom corner the message: "Plugin Update Recommended / Android Gradle Plugin is ready to update", clicked "update" -- and project works correctly!

[![printscrean02](https://user-images.githubusercontent.com/69940079/96642632-b4ec4480-1326-11eb-8539-fa8de98f5ad1.jpg)
](url)
